ci/build_setup.sh

```diff
@@ -136,7 +136,7 @@ cp -f "${ENVOY_SRCDIR}"/.bazelrc "${ENVOY_CI_DIR}"/
 # TODO(PiotrSikora): remove once we deprecate tools/bazel.rc in favor of .bazelrc.
 mkdir -p "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/tools
 mkdir -p "${ENVOY_CI_DIR}"/tools
-ln -sf "${ENVOY_SRCDIR}"/tools/bazel.rc "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/tools/
-ln -sf "${ENVOY_SRCDIR}"/tools/bazel.rc "${ENVOY_CI_DIR}"/tools/
+cp -f "${ENVOY_SRCDIR}"/tools/bazel.rc "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/tools/
```

Comment:
So I think this is the real fix and others are format fix?

Reply:
Yeah, look at the 2 commits. Honestly, I just wanted to push the first one, but then it was hard to verify that it actually fixes the issue on CircleCI.